service/src/test/java/uk/nhs/adaptors/gp2gp/utils/XmlParsingUtility.java

```diff
@@ -0,0 +1,48 @@
+package uk.nhs.adaptors.gp2gp.utils;
+
+import org.w3c.dom.Document;
+
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+public final class XmlParsingUtility {
+    private XmlParsingUtility() { }
+
+    public static Document getW3cDocumentFromXmlString(String xmlString) throws
```

Comment:
I think we can skip "W3c" from the name here (and other methods below), as it is mostly very specific to the implementation, 
something like "getDocumentFromXmlString" or "getXmlDocumentFromXmlStringStructure" etc